jetty-documentation/src/main/asciidoc/administration/startup/screen-http-webapp-deploy-listconfig.adoc

```diff
@@ -66,7 +66,6 @@ Version Information on 11 entries in the classpath.
 Note: order presented here is how they would appear on the classpath.
       changes to the --module=name command line options will be reflected here.
  0:                    3.1.0 | ${jetty.home}/lib/servlet-api-3.1.jar
- 1:                 3.1.0.M0 | ${jetty.home}/lib/jetty-schemas-3.1.jar
```

Comment:
All of the documentation is now wrong - it refers to servlet 3, but it needs to refer to servlet 4.

Reply:
Yup, but that documentation is wrong in `jetty-10.0.x` as well.
That can be a different commit / PR.